bin/cron/build_contact_rollups_v2

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+
+require_relative('../../dashboard/config/environment')
+require 'cdo/contact_rollups_v2'
+require 'cdo/contact_rollups_validation'
+require 'cdo/pardot'
+require 'cdo/chat_client'
+require 'cdo/only_one'
+require 'cdo/log_collector'
+require 'honeybadger/ruby'
+
+CHAT_CHANNEL = "cron-daily".freeze
+
+def main
+  log_collector = LogCollector.new "Build contact rollups"
+
+  # Build new daily table of contact rollups
+  ContactRollupsV2.build_contact_rollups(log_collector)
+end
```

Comment:
You can print out content of log_collector by `log_collector.to_s`. We can add error handling and Honeybadger/slack notification later if needed.